### PR TITLE
Update gsoc2023ideas.md

### PR DESCRIPTION
**Pull request by just-ary27**
Added changes for the BLT project.

### CODE REVIEW

**Comment by psiinon** [comment]
Lots of links to BugHeist specific resources rather than OWASP project ones...

**Comment by just-ary27** [comment]
Bug Logging Tool's MVP is essentially the BugHeist website and BugHeist app. Isn't then BugHeist an OWASP project?

**Comment by psiinon** [comment]
Superficially BugHeist looks like a separate (commercial?) entity built on top of OWASP BLT.
https://www.bugheist.com/ even says "Powered by OWASP Bug Logging Tool".
I dont think that makes BugHeist an OWAP project - there seems to be an important distinction here.
But I dont know the full background - is there anything that explains the relationship better?

**Comment by psiinon** [comment]
OK. This PR probably isnt the right sort of place to be discussing these things in too much detail :)
However I am going to raise it on the OWASP projects committee (which I'm a member of) as I'm still unsure of the exact relationship here...

**Comment by just-ary27** [comment]
Lemme know how that went :)

**Comment by just-ary27** [comment]
@psiinon Any updates?

**Comment by fredfalcon** [comment]
Bugheist is a separate commercial project using the OWASP BLT project through the  AGPLv3 open source license.  It was recommended by OWASP that we set it up this way and have the powered by BLT link on the bottom.  The reason we did it this way was that OWASP projects can’t pay the people that work on them through OWASP funds and we wanted the ability to pay people that worked on the project, essentially it’s a separation of finances.  So yes, when it comes to OWASP related correspondence, please use the official OWASP BLT name.  Otherwise, everything else is the same.

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 1]
```diff
@@ -94,16 +94,17 @@ All ZAP projects will be mentored by the ZAP Project Leaders:
 
 ### [Bug Logging Tool (BLT)](https://owasp.org/www-project-bug-logging-tool/)
 
+BugHeist is a _bug-hunting & logging_ tool which allows users and companies to hunt for bugs, claim bug bounties and also to start bug-hunting sprees/contests respectively. It is preferred if the potential GSoC contributors have atleast one PR merged into BugHeist app or BLT Core. For more info on the present initiatives check [this.](https://justary27.notion.site/BugHeist-834f6e7ae2f7489785cd6297bbc4d98f)
```
Comment (fredfalcon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:97]:
```suggestion
OWASP BLT is a _bug-hunting & logging_ tool which allows users and companies to hunt for bugs, claim bug bounties and also to start bug-hunting sprees/contests respectively. It is preferred if the potential GSoC contributors have atleast one PR merged into BLT. 
```

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 2]
```diff
@@ -94,16 +94,17 @@ All ZAP projects will be mentored by the ZAP Project Leaders:
...
+
 ![Preferred for "Medium" GSoC 2023 project](https://img.shields.io/badge/medium%20size%20(~175h)-preferred-green)
 ![Possible for "Large" GSoC 2023 project](https://img.shields.io/badge/large%20size%20(~350h)-possible-yellow)
 
 ##### Explanation of Ideas
 * Crypto - create a proof of stake coin called BUG
-* Flutter - ideas to improve the app? Launch it on the App Store.
+* Flutter - Check the [app section](https://justary27.notion.site/BugHeist-834f6e7ae2f7489785cd6297bbc4d98f) of initiatives page.
```
Comment (fredfalcon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:104]:
```suggestion
* Flutter - 
## App Specific Initiatives

- Add designs for a dark theme in the Figma file, then implement a generic theme manager for future themes.
- Start with app internationalization and localization (l10n).
- ***Design and build the company side part of the app.***
- ***Allow anonymous reporting*** for both logged-in and logged-out users, logged-in users should get an option to select whether to report a bug anonymously.
- ***Close most of the current issues on the app repo.***
- ***Bring the app to MVP level for both Android & iOS.***
```

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 3]
```diff
@@ -94,16 +94,17 @@ All ZAP projects will be mentored by the ZAP Project Leaders:
...
 * Django - Website integration of new design
 * Private issue reporting - allow companies to switch on private issue reporting.
 * Payments for issues reported - allow companies or individuals to pay big hunters
-* Browser plug-in to check bug reports - scan each site visited against a database to see if any bugs were found - we have a plug-in for chrome, let’s update it.
```
Comment (fredfalcon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:106]: Please add this back

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 4]
```diff
@@ -112,7 +113,7 @@ All ZAP projects will be mentored by the ZAP Project Leaders:
 ##### Getting Started
 
 * [BLT Core](https://github.com/owasp/blt/)
-* [Flutter App](https://github.com/Bugheist/Flutter/)
+* [BugHeist Mobile App](https://github.com/Bugheist/Flutter/)
```
Comment (fredfalcon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:116]:
```suggestion
* [Flutter App](https://github.com/Bugheist/Flutter/)
```

**Review by fredfalcon — CHANGES_REQUESTED** [review]

**Comment by fredfalcon** [comment]
@just-ary27 feel free to add the initiatives to our discussion group https://github.com/OWASP/BLT/discussions

**Comment by psiinon** [comment]
> @psiinon Any updates?

We're going to discuss this in the next Project Committee meeting, which is scheduled for 22 Feb.
I'm all for commercial use of OWASP tools but we have to keep a very careful separation.
Right now it doesnt look (to me) that this is happenning for BLT - there are way too many links and references to BugHeist, both in this PR and on the OWASP BLT pages.
I dont think this PR should be merged until all references to BugHeist are removed or replaces with BLT ones.
The close relationship between BLT and BugHeist is a wider issue that we will be address via the Projects Committee - I dont think that has to hold up this PR.

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 4 above)
Comment (fredfalcon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:116]:
```suggestion
```

**Review by fredfalcon — CHANGES_REQUESTED** [review]
Removing all references of Bugheist

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 4 above)
Comment (just-ary27) [reply pages/initiatives/gsoc/gsoc2023ideas.md:116]: I think we should keep a reference to the app under the previous "Flutter App" name.

Comment (fredfalcon) [reply pages/initiatives/gsoc/gsoc2023ideas.md:124]: I think we have to remove any reference of Bugheist even if it's in the URL, so lets take it out to be on the safe side.

Comment (just-ary27) [reply pages/initiatives/gsoc/gsoc2023ideas.md:124]: How will people know where to look for contributing on the flutter app if they want?

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 5]
```diff
@@ -121,7 +121,7 @@ You can chose to work on one or a combination of ideas depending on the difficul
 ##### Getting Started
 
 * [BLT Core](https://github.com/owasp/blt/)
-* [BugHeist Mobile App](https://github.com/Bugheist/Flutter/)
+* [Flutter App](https://github.com/Bugheist/Flutter/)
```
Comment (psiinon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:124]: The Flutter app looks to me like its just for Bugheist.
I dont think thats a suitable GSoC project.

Comment (just-ary27) [reply pages/initiatives/gsoc/gsoc2023ideas.md:124]: Flutter app has been in the GSoC ideas for BLT for the past 3 years with mentees contributing to it, additionally currently there's no commercial usage in BugHeist. Hence I think, it's a suitable one for GSoC. We also intend to remove BugHeist references in the app.

Comment (fredfalcon) [reply pages/initiatives/gsoc/gsoc2023ideas.md:124]: Yes we’re going to be correcting the app name to be BLT-Flutter and make it clear to contributors to use the BLT name.  Let’s do that first @just-ary27 and then we can add it back to the PR.  For now let’s take it out.

Comment (psiinon) [reply pages/initiatives/gsoc/gsoc2023ideas.md:124]: Cool, that sounds good!

Comment (just-ary27) [reply pages/initiatives/gsoc/gsoc2023ideas.md]: @psiinon Added the change, mergeable now.

**Review by fredfalcon — APPROVED** [review]

**Comment by just-ary27** [comment]
@psiinon Can you merge the PR, all the requested changes have been made. (Just a reminder)

**Comment by psiinon** [comment]
I'm ok with it now :)
Anyone else got any comments on it?

**Review by kingthorin — APPROVED** [review]

**Pull request merged**